### DISCUSSION TITLE
Unable to signup

### DISCUSSION THREAD

**Discussion by victoryosiobe** [npm/feedback#993] [Website] [answered]
I tried signing up on npm, put in everything right, then time for captcha. It takes a long while before showing a green tick. Now, create account button is accessible, but everytime I click on it. An error pops up on same page after reloading. Catpcha failed! I looked at the documentation from the error suggestion, passed all tests, but still same thing. The doc is not helping.

**Comment by monishcm** [comment, marked as answer]
Will look into this.
